docs/sources/alerting/monitor/meta-monitoring/_index.md

```diff
@@ -67,19 +83,31 @@ This metric is a histogram that shows you the number of seconds taken to send no
 
 > These metrics are not available at present in Grafana Cloud.
 
-## Grafana Mimir
+## Metrics for Mimir-managed alerts
+
+{{% admonition type="note" %}}
+This metric is available in OSS, on-premise, and Grafana Cloud.
```

Comment:
```suggestion
These metrics are available in OSS, on-premise, and Grafana Cloud.
```

Reply:
There is more than one metric in total, although for some reason I documented just one of them when I wrote this page. Not sure if we should change it from singular to plural in that case?